### PR TITLE
16.0 mig account banking sepa direct debit

### PR DESCRIPTION
**Pull request by gfcapalbo**
https://github.com/OCA/bank-payment/issues/964

WIP : testing,  migrated

### CODE REVIEW

File: account_banking_sepa_direct_debit/models/bank_payment_line.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        """
+        From my experience, even when you ask several direct debits
+        at the same date with enough delay, you will have several credits
+        on your bank statement: one for each mandate types.
+        So we split the transfer move lines by mandate type, so easier
+        reconciliation of the bank statement.
+        """
+        hashcode = super().move_line_offsetting_account_hashcode()
+        hashcode += "-" + str(self.mandate_id.recurrent_sequence_type)
+        return hashcode
```
Comment (Reyes4711-S73) [inline account_banking_sepa_direct_debit/models/bank_payment_line.py:20]:
```suggestion
```
In this PR https://github.com/OCA/bank-payment/pull/972, this model was removed

File: account_banking_sepa_direct_debit/models/account_payment_order.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+            group_header,
+            nb_of_transactions_a,
+            control_sum_a,
+        ) = self.generate_group_header_block(pain_root, gen_args)
+        transactions_count_a = 0
+        amount_control_sum_a = 0.0
+        lines_per_group = {}
+        # key = (requested_date, priority, sequence type)
+        # value = list of lines as objects
+        for line in self.bank_line_ids:
```
Comment (Reyes4711-S73) [inline account_banking_sepa_direct_debit/models/account_payment_order.py:75]: This field does not exist due to the fact that the module bank.payment.line was deleted. Please, change it

**Comment by Reyes4711-S73** [comment]
Functional review incorrect. There are some errors.
@pedrobaeza

**Comment by gfcapalbo** [comment]
@Reyes4711-S73 thanks for the review, will be checking your branch.

**Pull request closed without merging**